site/realm/data_sources/mongodb-atlas/aiidprod/checklists/schema.json

```diff
@@ -0,0 +1,57 @@
+ { 
+  "title": "checklists",
+  "properties": {
+    "_id":       { "bsonType": "objectId" },
+    "id":        { "bsonType": "string" },
+    "entity_id": { "bsonType": "string" },
+    "name":      { "bsonType": "string" },
+    "about":     { "bsonType": "string" },
+    "tags_goals": {
+      "bsonType": "array",
+      "items": {"bsonType": "string"}
+    },
+    "tags_methods": {
+      "bsonType": "array",
+      "items": {"bsonType": "string"}
+    },
+    "tags_other": {
+      "bsonType": "array",
+      "items": {"bsonType": "string"}
+    },
+    "risks": {
+      "bsonType": "array",
+      "items": {
+        "bsonType": "object",
+        "properties": {
+          "id":          { "bsonType": "string" },
+          "title":       { "bsonType": "string" },
+          "risk_status": { "bsonType": "string" },
+          "risk_notes":  { "bsonType": "string" },
+          "severity":    { "bsonType": "string" },
+          "likelihood":  { "bsonType": "string" },
+          "touched":     { "bsonType": "bool" },
+          "generated":   { "bsonType": "bool" },
+          "tags": { 
+            "bsonType": "array", 
+            "items": {"bsonType": "string"}
+          },
+          "precedents": {
+            "bsonType": "array", 
+            "items": {
+              "bsonType": "object",
+              "properties": {
+                "incident_id": {"bsonType": "int"},
+                "title":       {"bsonType": "string"},
+                "description": {"bsonType": "string"},
+                "tags": {
+                  "bsonType": "array",
+                  "items": { "bsonType": "string" }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
I'm still reading this closer and will come back to this.

Reply:
Resolving given that this is a prototype still and we will change schema and the collection if we need before v1.